x/airdrop/abci.go

```diff
@@ -9,6 +9,19 @@ import (
 
 // EndBlocker is called at the end of every block
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
+	params := k.GetParams(ctx)
+	if !params.OriginAccountsCreated {
+		airdropAccounts := k.GetAllAirdropAccounts(ctx)
+		for _, airdropAccount := range airdropAccounts {
+			err := k.CreateAirdropAccount(ctx, airdropAccount)
+			if err != nil {
+				ctx.Logger().Error("Error creating airdrop account", err)
+			}
+		}
```

Comment:
What's the performance impact here on the first block? If we have a million airdrop accounts I wonder what it'd look like.

Maybe we should chunk this out over time.

Reply:
How do you suggest implementing that? Create X number of account each block until complete?

Reply:
Updated to add batch processing.